docs/application/tizen-studio/release-notes/3-2-release-notes.md

```diff
@@ -0,0 +1,71 @@
+# Tizen Studio 3.2 Release Notes
+
+-   Release Date: Feb 20, 2019
+
+
+## IDE and Tools
+
+
+### New Features
+-   Tizen Studio
+	-  Added New watch application template for 5.0 Wearable.
+	-  Added New Skin for Wearable Emulator without bezel to disable rotation.
+
+### Fixed Bugs
+-   CLI
+	-   Fixed the strip option in package command, it was not working as expected.
+-   Certificate manager
+	-   Improved package signing time by reducing the loading of unnecessary security profiles.
+-   BaseLine SDK
+	-   Updated netcoredbg package to support debugging on 5.0 Emulator.
+
+### Known Issues
+-	Tizen Studio
+	-	In MacOS, if the UI perspectives are not displayed properly after updating the Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from previous installation.
+	-	In case if you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+-	Common
+	-   If you install the Tizen Studio in a directory that requires administrator privileges for access. For example, for C:\ProgramFiles, administrator privileges are required to run the Tizen SDK tools. The Tizen Installer and the baseline SDK Installer alerts you, if you try to install into such a directory.
+-	Web and Native IDE
+	-   From Tizen Studio 2.0 onwards, the Connection Explorer is replaced with the Device Manager, this can cause errors in the Connection Explorer view. You can fix this in two ways:
+	    -   Reset the perspective.
+            In the Tizen Studio menu, select Window > Perspective > Reset Perspective
```

Comment:
Global Comment:
Please remove "the" from "The Tizen Studio".
Applicable to line no: 24, 27, 32, 33, 46, 60.

Reply:
Fixed, thanks for the review